File: Programming-languages-and-laboratory/Python/Exercises/09_compito_java_20160615/Basket.py
from Squadra import Squadra


class Basket(Squadra):
    def __init__(self, nome, codice, sport, vinte, perse, punteggio_medio):
        super().__init__(nome, codice, sport, vinte, perse)
        self._punteggio_medio = punteggio_medio

    def __str__(self):
        return super().__str__() + "\t" + str(self._punteggio_medio) + "\t-\t" + self._sport

File: Programming-languages-and-laboratory/Java/Exercises/09_compito_java_2016_06_15/Basket.java
public class Basket extends Squadra {
    private float punteggioMedio;

    public Basket(String nome, int codice, String sport, int vinte, int perse, float punteggioMedio) {
        super(nome, codice, sport, vinte, perse);
        this.punteggioMedio = punteggioMedio;
    }

    public String toString() {
        return super.toString() + "\t-\t" + punteggioMedio + "\t" + "basket";
    }
}